yarn.lock

```diff
@@ -16,39 +16,38 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
-"@babel/compat-data@^7.13.0", "@babel/compat-data@^7.13.12", "@babel/compat-data@^7.13.8":
-  version "7.13.12"
-  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.13.12.tgz#a8a5ccac19c200f9dd49624cac6e19d7be1236a1"
-  integrity sha512-3eJJ841uKxeV8dcN/2yGEUy+RfgQspPEgQat85umsE1rotuquQ2AbIub4S6j7c50a2d+4myc+zSlnXeIHrOnhQ==
+"@babel/compat-data@^7.13.11", "@babel/compat-data@^7.13.15", "@babel/compat-data@^7.13.8", "@babel/compat-data@^7.14.0":
```

Comment:
Are the changes to the `yarn.lock` necessary to the pull-request?

Reply:
Inversify seems to have made incompatible changes recently. So when I compile against a yarn-linked version of theia source, I get a different version of the inversify module that is not source-compatible with the currently resolved one. Aka...the modules depending on theia don't compile. So while it's not related to this particular change, we should update the yarn.lock

Reply:
@tsmaeder please see https://github.com/eclipse-theia/theia/pull/9413, we generally try to do a `yarn upgrade` following releases to precisely update the `yarn.lock` to reflect the state of what downstream applications might pull. You're correct that some changes were made to inversify.

Reply:
Cool, once you've merged that one, I can test again.